source/features/upload-button.js

```diff
@@ -3,22 +3,42 @@ import select from 'select-dom';
 import delegate from 'delegate';
 import {metaKey} from '../libs/utils';
 import * as icons from '../libs/icons';
+import observeEl from '../libs/simplified-element-observer';
 
-function addButtons() {
+let delegations;
+
+async function addButtons() {
 	for (const form of select.all('.js-previewable-comment-form:not(.rgh-has-upload-field)')) {
 		if (!select.exists('.js-manual-file-chooser[type=file]', form)) {
 			continue;
 		}
-		form.classList.add('rgh-has-upload-field');
-		const toolbarPosition = select('.js-saved-reply-container', form);
-		if (!toolbarPosition) {
-			continue;
-		}
-		toolbarPosition.after(
-			<button type="button" class="toolbar-item rgh-upload-btn">
-				{icons.cloudUpload()}
-			</button>
-		);
+
+		const toolbar = select('markdown-toolbar', form);
+
+		const observer = observeEl(toolbar, () => {
+			if (form.classList.contains('rgh-has-upload-field')) {
+				observer.disconnect();
```

Comment:
This should also be called after the first run, like after line 37

Reply:
Answered [above](https://github.com/sindresorhus/refined-github/pull/1262#discussion_r181586751).